### PR TITLE
MAILBOX-331 Fix Eclipse warnings

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: mailbox/plugin/quota-mailing/src/main/java/org/apache/james/eventsourcing/CommandDispatcher.java [hunk 1]
```diff
@@ -78,10 +78,11 @@ public int getRetries() {
     }
 
     private final EventBus eventBus;
+    @SuppressWarnings("rawtypes")
     private final Map<Class, CommandHandler> handlers;
```
Comment (chibenwa) [inline mailbox/plugin/quota-mailing/src/main/java/org/apache/james/eventsourcing/CommandDispatcher.java]: Can't a `private final Map<Class, CommandHandler<?>> handlers;` do the job as well?

Comment (aduprat) [reply mailbox/plugin/quota-mailing/src/main/java/org/apache/james/eventsourcing/CommandDispatcher.java]: Then I'm failing with a _capture-of_ in the _tryDispatch_ method.
BTW, we still need it for _Class<?>_

**Comment by aduprat** [comment]
```

Tests in error: 
  SpamAssassinContract.imapCopiesToSpamMailboxShouldBeConsideredAsSpam:181 » ConditionTimeout
  SpamAssassinContract.imapMovesToSpamMailboxShouldBeConsideredAsSpam:228 » ConditionTimeout
```
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**